docs/faq.md

```diff
@@ -20,15 +20,42 @@ Skin can be simulated using the [AdnSkin](maya/skin) deformer. This deformer req
 
 Similarly to a skin simulation setup, the fascia can be simulated using [AdnSkin](maya/skin) deformer as well. In this case, it is recommended to use values of *Rest Length Multiplier* lower than 1. This deformer also requires one mesh (a single target, i.e. the muscle geometries combined) or multiple meshes (multiple targets, i.e. the muscle geometries separated) to drive the simulation and a mesh to apply the deformer to (fascia geometry).
 
-## Can I simulate muscles, fascia and skin all coupled?
+## How can I generate the fascia geometry?
 
-Yes, you can simulate muscles, fascia and skin following these steps:
+In order to obtain a plausible fascia geometry, we can use two techniques that require the muscle geometries and the skin geometry as inputs.
+
+The easiest option is to use the shrinkwrap deformer provided by Maya. This solution works better for low poly geometries.
+
+- Create a copy of the skin geometry that will become your fascia.
+- Select the combined geometry of all muscles, then select the copy of the skin geometry.
```

Comment:
Split into 2 steps please:
- Combined all the muscles together
- Select the combined muscles geometry, then select the copy of the skin geometry.

Reply:
Done.